purchase_order_line_initial_quantity/models/purchase_order.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    initial_qty_changed = fields.Boolean(
```

Comment:
Hi, I'm not sure to understand the use case behind this module.
AFAIK, once a PO is validated, the ordered quantities are not available for change so the qunatities are the original ones.
Any feedback could help, maybe in the readme file this could help